compiler/src/dotty/tools/dotc/transform/Pickler.scala

```diff
@@ -194,13 +289,27 @@ class Pickler extends Phase {
   }
 
   override def runOn(units: List[CompilationUnit])(using Context): List[CompilationUnit] = {
+    val isConcurrent = useExecutor
```

Comment:
If this is nothing but a local cache of the `useExecutor` method call, consider naming it the same:
```suggestion
    val useExecutor = this.useExecutor
```